src/platform/posix/split.rs

```diff
@@ -68,12 +68,39 @@ pub struct Reader {
     pub(crate) fd: Arc<Fd>,
     pub(crate) offset: usize,
     pub(crate) buf: Vec<u8>,
+    pub(crate) mtu: u16,
 }
 
 impl Reader {
     pub(crate) fn set_mtu(&mut self, value: u16) {
+        self.mtu = value;
         self.buf.resize(value as usize + self.offset, 0);
     }
+
+    pub(crate) fn recv(&self, mut buf: &mut [u8]) -> io::Result<usize> {
+        let mut stack_buf = [0u8; crate::DEFAULT_MTU as usize + PIL];
```

Comment:
Based on we want to do some optimization here, let us move the `stack_buf` to line 90 here to avoid each useless initialization of `stack_buf` every time we enter `recv` without using `stack_buf`. The same is true for `send`.

Reply:
Sounds good. Updated